.github/workflows/trunk.yml

```diff
@@ -264,3 +264,14 @@ jobs:
       docker-image: ${{ needs.linux-focal-cuda12_4-py3_10-gcc9-experimental-split-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-focal-cuda12_4-py3_10-gcc9-experimental-split-build.outputs.test-matrix }}
     secrets: inherit
+
+  # NB: Keep this in sync with inductor-perf-test-nightly.yml
+  linux-focal-cuda12_1-py3_10-gcc9-inductor-build:
```

Comment:
CUDA-12.1 again... Noooo
@atalman weren't migration over?

Reply:
@malfet  sorry missed this. Yes this should be 12.4 you totally right

Reply:
The performance benchmarks are still on 12.4. So you need to migrate the perf benchmarks...